bentoml/_internal/service/service.py

```diff
@@ -161,6 +219,7 @@ def mount_asgi_app(
     def mount_wsgi_app(
         self, app: WSGI_APP, path: str = "/", name: t.Optional[str] = None
     ) -> None:
+        # TODO: Migrate to a2wsgi
```

Comment:
https://github.com/abersheeran/a2wsgi is this the library @parano ?

Reply:
I wonder why do we do that

Reply:
@bojiang it's a warning from starlette that it's deprecating this API

Reply:
That lib is not using anyio. I thought we should do more investigation. or even wrote ourselves